rascal-textmate-core/src/main/rascal/util/MaybeUtil.rsc

```diff
@@ -0,0 +1,28 @@
+@synopsis{
```

Comment:
The functions in this module were previously part of module `lang::rascal::grammar::analyze::Symbols`, but they're now needed also in other places